embassy-stm32/src/timer/mod.rs

```diff
@@ -438,11 +438,6 @@ pub(crate) mod sealed {
         fn set_dead_time_value(&self, value: u8) {
             Self::regs_1ch_cmp().bdtr().modify(|w| w.set_dtg(value));
         }
-
-        /// Enable timer outputs.
-        fn enable_outputs(&self) {
-            Self::regs_1ch_cmp().bdtr().modify(|w| w.set_moe(true));
-        }
```

Comment:
could you restore this method? `MOE` existed on all `TIM_1CH_CMP` `TIM_2CH_CMP` and `TIM_ADV`.

Reply:
Or even better, rewrite this method to something like `fn set_moe(&self, enable: bool)`, and call this method in public trait implement.

Reply:
Replaced `enable_outputs()`by `fn set_moe(&self, enable: bool)`